src/tree-view/__tests__/tree-item.test.tsx

```diff
@@ -0,0 +1,115 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as React from 'react';
+import { render } from '@testing-library/react';
+
+import createWrapper from '../../../lib/components/test-utils/dom';
+import TreeView, { TreeViewProps } from '../../../lib/components/tree-view';
+
+interface Item {
+  id: string;
+  title: string;
+  items?: Item[];
+}
+
+const defaultItems: Item[] = [
+  {
+    id: '1',
+    title: 'Item 1',
+    items: [
+      {
+        id: '1.1',
+        title: 'Item 1.1',
+      },
+      {
+        id: '1.2',
+        title: 'Item 1.2',
+        items: [
+          {
+            id: '1.2.1',
+            title: 'Item 1.2.1',
+          },
+        ],
+      },
+    ],
+  },
+  {
+    id: '2',
+    title: 'Item 2',
+  },
+];
+
+const defaultProps: TreeViewProps<Item> = {
+  items: defaultItems,
+  getItemId: item => item.id,
+  getItemChildren: item => item.items,
+  renderItem: item => ({
+    content: item.title,
+  }),
+};
+
+function renderTreeView(props: Partial<TreeViewProps<Item>> = {}) {
```

Comment:
nit: there are common functions and default props across tree view test files - would it make sense to move those to some common.ts?